spec/workflows/end_product_code_selector_spec.rb

```diff
@@ -0,0 +1,130 @@
+  %w[040BDENRPMC primary na dta pension nonrating appeal],
+  %w[040BDER primary na dta compensation rating appeal],
+  %w[040BDERPMC primary na dta pension rating appeal],
+  %w[040HDENR primary na dta compensation nonrating higher_level_review],
+  %w[040HDENRPMC primary na dta pension nonrating higher_level_review],
+  %w[040HDER primary na dta compensation rating higher_level_review],
+  %w[040HDERPMC primary na dta pension rating higher_level_review],
+  %w[930AMAHDENCL 930 local_quality_error dta compensation nonrating higher_level_review],
+  %w[930AMAHDENCN 930 national_quality_error dta compensation nonrating higher_level_review],
+  %w[930AMAHDENR 930 control dta compensation nonrating higher_level_review],
+  %w[930AMAHDER 930 control dta compensation rating higher_level_review],
+  %w[930AMAHDERCL 930 local_quality_error dta compensation rating higher_level_review],
+  %w[930AMAHDERCN 930 national_quality_error dta compensation rating higher_level_review],
+  %w[930AMARNRC 930 control dta compensation nonrating appeal],
+  %w[930AMARRC 930 control dta compensation rating appeal],
+  %w[930AMARRCLQE 930 local_quality_error dta compensation rating appeal],
+  %w[930AMARRCNQE 930 national_quality_error dta compensation rating appeal],
+  %w[930AMARRCPMC 930 control dta pension rating appeal],
+  %w[930ARNRCLPMC 930 local_quality_error dta pension nonrating appeal],
+  %w[930ARNRCLQE 930 local_quality_error dta compensation nonrating appeal],
+  %w[930ARNRCNPMC 930 national_quality_error dta pension nonrating appeal],
+  %w[930ARNRCNQE 930 national_quality_error dta compensation nonrating appeal],
+  %w[930ARNRCPMC 930 control dta pension nonrating appeal],
+  %w[930ARRCLQPMC 930 local_quality_error dta pension rating appeal],
+  %w[930ARRCNQPMC 930 national_quality_error dta pension rating appeal]
+].freeze
+
+describe "Request Issue Correction Cleaner", :postgres do
+  let(:higher_level_review) { create(:higher_level_review, benefit_type: benefit_type) }
+  let(:supplemental_claim) { create(:supplemental_claim, benefit_type: benefit_type, decision_review_remanded: drr) }
+  let(:appeal) { create(:appeal) }
+  let(:benefit_type) { nil }
+  let(:correction_type) { nil }
+  let(:decision_review) { nil }
+  let(:drr) { nil }
+  let(:contested_rating_issue_reference_id) { nil }
+  let(:nonrating_issue_category) { nil }
+  let(:is_unidentified) { nil }
+  let(:request_issue) do
+    create(:request_issue,
+           nonrating_issue_category: nonrating_issue_category,
+           contested_rating_issue_reference_id: contested_rating_issue_reference_id,
+           decision_review: decision_review,
+           benefit_type: benefit_type,
+           correction_type: correction_type,
+           is_unidentified: is_unidentified)
+  end
+
+  context "#call" do
+    subject { EndProductCodeSelector.new(request_issue).call }
+
+    context "for an unidentified issue" do
+      let(:is_unidentified) { TRUE }
```

Comment:
```suggestion
      let(:is_unidentified) { true }
```

I get a warning on this line: `warning: constant ::TRUE is deprecated`